core/src/main/java/io/questdb/log/LogFactory.java

```diff
@@ -561,6 +457,39 @@ private static void setLogger(Class<?> clazz, Log logger) {
         }
     }
 
+    private void configure(InputStream fis, String rootDir) throws IOException {
```

Comment:
this method used to be static and this PR removed the following lines:

```
                                if (logDir != null && value.contains(LOG_DIR_VAR)) {
                                    value = value.replace(LOG_DIR_VAR, logDir);
                                }
```

which were introduced in https://github.com/questdb/questdb/pull/1969

why were they removed?